src/DotNetWorker.ApplicationInsights/DotNetWorker.ApplicationInsights.csproj

```diff
@@ -8,17 +8,14 @@
     <MajorProductVersion>1</MajorProductVersion>
     <MinorProductVersion>0</MinorProductVersion>
     <PatchProductVersion>0</PatchProductVersion>
-    <VersionSuffix>-preview1</VersionSuffix>
+    <VersionSuffix>-preview2</VersionSuffix>
   </PropertyGroup>
 
   <Import Project="..\..\build\Common.props" />
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Core" Version="1.7.0-preview1" />
```

Comment:
Curious to know why we switched to package reference? Is it to lock this version of AI package to specific version of Worker.Core package?

Reply:
Yeah, there's a small tweak in preview1 that we need for this package to work... but there's no need for me to keep revving that one now that I don't need additional changes. It was mostly a local dev improvement to lock this to a published package.

This will eventually go away and back to a proj reference once this is ready for GA.